ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -288,8 +289,8 @@ private void planTasks() {
     }
   }
 
-  private double getAvailableCore(TableRuntime tableRuntime) {
-    return tableRuntime.getOptimizingConfig().getTargetQuota();
+  private double getAvailableCore() {
+    return authOptimizers.values().stream().map(Resource::getThreadCount).reduce(Integer::sum).orElse(0);
```

Comment:
Maybe we should match table group ?

Reply:
Do you mean the `Optimizer Group`? All the `authOptimizers` have the same `Optimizer Group` in the object of `OptimizingQueue`.

Reply:
Yes, you are right. My fault.